src/main/java/org/sagebionetworks/bridge/exporter/worker/TsvInfo.java

```diff
@@ -49,11 +51,12 @@ private TsvInfo() {
      * @param writer
      *         writer for the TSV file
      */
-    public TsvInfo(List<String> columnNameList, File file, PrintWriter writer) {
+    public TsvInfo(List<String> columnNameList, File file, PrintWriter writer, List<String> recordIds) {
         this.columnNameList = columnNameList;
         this.file = file;
         this.writer = writer;
         this.initError = false;
+        this.recordIds = recordIds;
```

Comment:
recordIds is always an empty list, so instead of having the constructors create it, we should initialize it in the declaration above.